spec/lib/tax_return_status_spec.rb

```diff
@@ -45,4 +45,23 @@
       end
     end
   end
+
+  context ".available_statuses_for" do
+    context "as a greeter" do
+      let(:greeter) { create :greeter_user }
+      it "only provides me with intake statuses" do
+        result = described_class.available_statuses_for(greeter)
+        expect(result.keys.length).to eq 1
+        expect(result.keys.first).to eq "intake"
```

Comment:
Minor thought: Another way we've written similar assertions is to use `match_array`
```ruby
expect(result.keys).to match_array ["intake"]
```